modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -855,19 +860,22 @@ public CompletableFuture<Ignite> start(Path configPath) {
                                     clientHandlerModule,
                                     deploymentManager,
                                     sql
-                            );
+                            ));
 
                             // The system view manager comes last because other components
                             // must register system views before it starts.
-                            lifecycleManager.startComponent(systemViewManager);
+                            startFuturesAfterJoin.add(lifecycleManager.startComponent(systemViewManager));
```

Comment:
Why isn't this component simply put add the end of the list above?

Reply:
to be honest, I didn't understand your proposal

Reply:
ok, I haven't thought about the comment, so your solution confused me